common/utils.go

```diff
@@ -105,3 +105,13 @@ func GetMevBoostVersionFromUserAgent(ua string) string {
 	}
 	return "-"
 }
+
+func reverse(src []byte) []byte {
```

Comment:
We should have a specific conversion function big.Int to uint256 that can also be properly tested.
It can use the reverse of course, but we absolutely need unit tests for this logic separately for reverse, and even more important, for the conversion itself.

Reply:
yeah agree, more tests would be nice, as well as specific conversion function! 🙏